tools/lan_choose_character_kart/asm/fielddefinitions.inc

```diff
@@ -0,0 +1,203 @@
+.equ kartProgressArr, 0x304 /*  array of 8 bytes */
+.equ rememberedSelections, 0x30c /*  array of 24 (3*8) bytes */
+.equ randSeed, 0x324 /*  word */
+.equ race2DArchive, 0x328 /*  word */
+.equ j2dPicture, 0x32c /*  word */
+.equ j2dPrintForFont, 0x330 /*  word */
+.equ charBackgroundPictureEx, 0x334 /*  word */
+.equ selectionsHaveBeenUsed, 0x338 /*  byte */
+.equ UnrestrictedModeSet, 0x339 /*  byte */
+
+/* --------------- */
+/*  KartPad fields */
+/* --------------- */
+.equ buttonHold, 0x18
+.equ buttonRepeat, 0x30
+.equ trigger, 0x1c
+.equ triggerLeft, 0x1
+.equ triggerRight, 0x2
+
+/* --------------- */
+/*  RaceInfo field */
+/* --------------- */
+.equ gRaceInfo_kartNumber, 0x1c /*  halfword */
+.equ kartInfos, 0x30 
+
+/* ---------------- */
+/*  KartInfo fields */
+/* ---------------- */
+.equ kartDB, 0x0
+.equ pad1, 0x4
+.equ char1DB, 0x8
+.equ pad2, 0xc
+.equ char2DB, 0x10
+.equ kartInfoTotalSize, 0x18
+
+/* ---------- */
+/*  DB fields */
+/* ---------- */
+.equ charDBID, 0x0
+.equ kartDBID, 0x0
+.equ partnerCharDB, 0x2
+.equ charDBWeight, 0x4
+.equ charKartDB, 0x6
+
+/* ------------------ */
+/*  JFWDisplay fields */
+/* ------------------ */
+.equ vt, 0x0
+.equ fader, 0x4
+.equ startFadeOut, 0x10
+
+.equ ParadeKartIdx, 20
+
+
+/* ----------------- */
+/*  NetGateApp field */
+/* ----------------- */
+.equ printMemoryCard, 0x2c
+.equ appHeap, 0x4
+
+/* ----------------- */
+/*  NetGameMgr field */
+/* ----------------- */
+.equ netRandSeed, 0x12e9
+
+/* ---------------------- */
+/*  PrintMemoryCard field */
+/* ---------------------- */
+.equ printWindow, 0x8
+
+/* ------------------ */
+/* PrintWindow fields */
+/* ------------------ */
+.equ windowSize, 0x2e8 /*  word */
+.equ windowColour, 0x2f0 /*  word */
+
+/*  J2DTextBox field */
+.equ stringPtr, 0x124
+
+/*  J2DPane fields */
+.equ sizeX, 0x28
+.equ sizeY, 0x2c
+.equ scaleX, 0xcc
+.equ scaleY, 0xd0
+.equ offsetX, 0xd4
+.equ offsetY, 0xd8
+.equ cornerColourTopLeft, 0x150
+.equ cornerColourTopRight, 0x154
+.equ cornerColourBottomLeft, 0x158
+.equ cornerColourBottomRight, 0x15c
+
+/* J2DPrint field */
+.equ fontSizeWidth, 0x50
+.equ fontSizeHeight, 0x54
+
+/* J2DPictureEx fields */
+.equ j2DMaterial, 0x168
+
+# J2DMaterial field
+.equ j2dTevBlock, 0x70
+
+/* J2DTevBlock vt methods */
+.equ j2dTevBlock_initiailize, 0x8
+
+
+/* Kart2DCommon field */
+.equ chara_window_1_bti, 0x140
+.equ characterPortraits_Bti, 0x94
+.equ kartPortraits_Bti, 0xe8
+
+# NetGateApp field
+.equ arrowBtiPtr, 0x34
+
+.equ rButton, 0x0020
+.equ lButton, 0x0040
+.equ zButton, 0x0010
+.equ aButton, 0x0100
+.equ bButton, 0x0200
+.equ xButton, 0x0400
+.equ startButton, 0x1000
+
+.equ GET_BUTTON, 0x0
+.equ GET_STICK, 0x1
+
+.equ PARAM_LEFTINPUT, -1
+.equ PARAM_RIGHTINPUT, +1
+
+.equ MENUPROGRESS_WAITFORENTRYTIMER, 4
+.equ MENUPROGRESS_INIT, 5
+.equ MENUPROGRESS_WAITFOROPENINGANIM, 6
+.equ MENUPROGRESS_CANMAKESELECTIONS, 7
+.equ MENUPROGRESS_RACEBATTLEWAIT, 8
+.equ MENUPROGRESS_BACKTOTITLEWAIT, 9 
```

Comment:
Nit: If you care, there are some trailing spaces in these files that could be stripped.

IDEs normally have options or extensions to show trailing spaces; I personally can't live without being able to see them.